frontend/src/pages/NewRunV2.tsx

```diff
@@ -155,12 +209,23 @@ function NewRunV2(props: NewRunV2Props) {
 
   // Handle different change that can affect setIsStartButtonEnabled
   useEffect(() => {
-    if (!templateString || errorMessage || !isParameterValid) {
+    if (
+      !templateString ||
+      errorMessage ||
+      !isParameterValid ||
+      !(apiPipelineVersion || (apiResourceRefFromRun && apiResourceRefFromRun[1]))
```

Comment:
Currently disabled start button for SDK-created run (no pipeline version, no apiResourceReference)

Reply:
Consider adding TODOs like this to the code itself.